### DISCUSSION TITLE
interview/kafka-40

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#232] [Announcements]
# interview/kafka-40

Java程序员进阶之路，小白的零基础Java教程，40 道 Kafka 精选面试题👍

https://tobebetterjavaer.com/interview/kafka-40.html

**Comment by debugzhao** [comment]
整理这么好的文章，竟然没有评论？

**Comment by xionghengheng** [comment]
问得最多的重平衡流程 没有啊

**Comment by swiftlc** [comment]
1. rebalence
2. 消息事务
3. 消息交付语义
